src/domain/map/hooks/useFeaturesAtPixel.ts

```diff
@@ -0,0 +1,45 @@
+import { useContext, useEffect, useState } from 'react';
+import { debounce } from 'lodash';
+import { MapBrowserEvent } from 'ol';
+import { FeatureLike } from 'ol/Feature';
+import { Pixel } from 'ol/pixel';
+import MapContext from '../../../common/components/map/MapContext';
+import { MapInstance } from '../../../common/components/map/types';
+
+export interface FeatureWithPixel {
+  feature: FeatureLike;
+  featurePixel: Pixel;
+}
+
+export default function useFeaturesAtPixel() {
```

Comment:
Maybe some comment here would be good as it is difficult to understand what this hook does just from the name

Reply:
Comment added.